File: scripts/corpus/known-benign/normal.js
const express = require('express')
const app = express()
const port = process.env.PORT || 3000

app.get('/', function(req, res) {
  res.json({ status: 'ok', message: 'Hello world' })
})

app.get('/health', function(req, res) {
  res.status(200).end()
})

app.listen(port, function() {
  console.log('Server running on port ' + port)
})

module.exports = app

File: scripts/corpus/known-benign/normal.py
from flask import Flask, jsonify, request

app = Flask(__name__)

@app.route('/')
def index():
    return jsonify({'status': 'ok', 'message': 'Hello world'})

@app.route('/api/data', methods=['GET'])
def get_data():
    query = request.args.get('q', '')
    return jsonify({'query': query, 'results': []})

if __name__ == '__main__':
    app.run(host='0.0.0.0', port=5000, debug=False)
